### PR TITLE
[DE] osu!stream

### PR DESCRIPTION
**Pull request by Firepe**

### CODE REVIEW

File: wiki/osu!stream/de.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
+<! - wikilink ->
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
<!-- wikilink -->
```
This is a comment, these must start with `<!--` and end with `-->`

File: wiki/osu!stream/de.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
+<! - wikilink ->
+[Songliste]: ./Song_List "Songliste"
+[NF wikilink]: / wiki / Game_Modifiers / "Weitere Informationen finden Sie in Game Modifiers unter No Fail"
+
+<! - img ->
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
<!-- img -->
```

File: wiki/osu!stream/de.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
+<! - wikilink ->
+[Songliste]: ./Song_List "Songliste"
+[NF wikilink]: / wiki / Game_Modifiers / "Weitere Informationen finden Sie in Game Modifiers unter No Fail"
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
[NF wikilink]: /wiki/Game_Modifiers/ "Weitere Informationen finden Sie in Game Modifiers unter No Fail"
```
This is a link, the path must not have any spaces

File: wiki/osu!stream/de.md [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+  - Es wird empfohlen, zuerst Pakete herunterzuladen, die kein Geld kosten. Gratis-Packs haben anstelle des "USD $" -Kosten-Tags "FREE".
+
+## Was ist der Unterschied?
+
+- **Nur auf iOS-Geräten spielbar.** (vorerst)
+- Beatmaps werden von Grund auf neu erstellt.
+- Zum Herunterladen der Song-Packs ist kein Konto erforderlich.
+- Kann osu! Stream mit einem Twitter-Account integrieren.
+- Der Highscore kann anonym unter dem Namen "Gast" an die Site gesendet werden (falls dies vorkommt, jedoch eine Internetverbindung erforderlich ist).
+- Die Kreisuhr wird ausgetauscht und mit einer kleinen Randlinie nach unten verschoben, die je nach Genauigkeit farbig ist. \ [Bottom (Long)\]
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
- Die Kreisuhr wird ausgetauscht und mit einer kleinen Randlinie nach unten verschoben, die je nach Genauigkeit farbig ist. \[Bottom (Long)\]
```

The `[` character is being escaped with the `\\` character, there cannot be a space in between the two characters.

File: wiki/osu!stream/de.md [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+![Finger Guides anzeigen][Os-FG.jpg]
+
+_Finger Guides display_
+
+- Der zwei durchsichtige Kreis, der im Spiel angezeigt wird, soll Anfängern helfen, zu wissen, wann sie die Objekte treffen müssen.
+  - Es zeigt die vorgeschlagene Methode, um die de design der Karte mit Perfektion.
+- Die grüne Farbe ist für die linke und die rote für die rechte Hand.
+- Diese Funktion kann aus dem Optionsmenü oder nach Abschluss des Tutorials deaktiviert werden.
+
+######Connected Beats
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
#### Connected Beats
```
The heading level in the EN version is 4. Also, a space is required between the `#` and text

File: wiki/osu!stream/de.md [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+- Halte den Kreis so lange, bis der Kreis "explodiert
+- 300 wenn perfekt gemacht.
+  - 100 für einen ungenaueren anfänglichen Halt und ein korrektes Halten.
+  - Zumindest 50 für das Berühren.
+  - 30 für jede Explosion.
+  - Multiplikator-Reset, wenn die Kreise nicht richtig gehalten wurden, und Miss, wenn sie dies nicht taten.
+
+**Sliders**
+
+- ** Die Beurteilung von Schiebereglern hängt von der Genauigkeit des anfänglichen Abgriffs ab **.
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
- **Die Beurteilung von Schiebereglern hängt von der Genauigkeit des anfänglichen Abgriffs ab**.
```

Do not put a space after the first `**` and before the last `**`

File: wiki/osu!stream/de.md [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+- Schieberegler bestehen aus Schieberegler-Häkchen, die die Start-, End- und Wiederholungspunkte des Schiebereglers enthalten.
+  - Ein 300er, wenn alle Slider-Ticks und der genaue erste Tipp erfolgreich ausgeführt wurden.
+  - Eine 100, wenn mindestens die Hälfte von ihnen mit erheblichem Anfangsabgriff.
+  - Eine 50, wenn es gelungen ist, auch nur einen der Ticks oder den ersten Tipp zu treffen.
+- ** Ein zu früher Klick auf einen Schieberegler ** führt nicht zu einem Fehlschlag, ** setzt jedoch den Punktemultiplikator auf 0 zurück **.
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
- **Ein zu früher Klick auf einen Schieberegler** führt nicht zu einem Fehlschlag, **setzt jedoch den Punktemultiplikator auf 0 zurück**.
```

File: wiki/osu!stream/de.md [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+
+_Hold circle 2_
+
+- Halte den Kreis so lange, bis der Kreis "explodiert
+- 300 wenn perfekt gemacht.
+  - 100 für einen ungenaueren anfänglichen Halt und ein korrektes Halten.
+  - Zumindest 50 für das Berühren.
+  - 30 für jede Explosion.
+  - Multiplikator-Reset, wenn die Kreise nicht richtig gehalten wurden, und Miss, wenn sie dies nicht taten.
+
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]: It looks like a bunch of content is missing around here.

From `New Modes to replace the difficulty` to half of `Score`

File: wiki/osu!stream/de.md [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+![Hold circle 2][OS-HC2.jpg]
+
+_Hold circle 2_
+
+- Halte den Kreis so lange, bis der Kreis "explodiert
+- 300 wenn perfekt gemacht.
+  - 100 für einen ungenaueren anfänglichen Halt und ein korrektes Halten.
+  - Zumindest 50 für das Berühren.
+  - 30 für jede Explosion.
+  - Multiplikator-Reset, wenn die Kreise nicht richtig gehalten wurden, und Miss, wenn sie dies nicht taten.
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]: It looks like the content here is different than the English one.

File: wiki/osu!stream/de.md [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+- Die Anzeige eines Spinners mit einem eindeutigen Status kann nicht aufgefüllt werden.
+
+Folgendes erhöht oder setzt den Punktemultiplikator nicht zurück:
+
+- Fehlendes Slider-Ende. (Wird zu einem guten Ergebnis führen! Oder 100/50 Treffer)
+- Spinner's Score Bonus
+
+### Noten
+
+! [Notenbildschirm] [Os-Grade.jpg]
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
![Notenbildschirm][Os-Grade.jpg]
```
Do not put spaces between `!` and `[` and between `]` and `[`. This is part of markdown.

File: wiki/osu!stream/de.md [hunk 11]
```diff
@@ -0,0 +1,254 @@
...
+- C = Über 60% Genauigkeit
+- D = Sonst noch was.
+
+### Lebensdaueranzeige
+
+Im Gegensatz zum osu! -Standard unterscheidet sich die Lebensleiste in osu! streamen, da es auf dem Gameplay-Modus basiert.
+
+Im **einfachen Modus wird die Lebensleiste vollständig entfernt**.
+
+![Beispiel für Stream Up! von Normal][Bild hoch streamen]
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
![Beispiel für Stream Up! von Normal][Bild streamen]
```

This is a reference link, you have translated the text on the right twice but differently (from line 11). The two must match to work correctly.

File: wiki/osu!stream/de.md [hunk 12]
```diff
@@ -0,0 +1,254 @@
...
+
+Im ** Stream-Modus ** erhält der Spieler jedoch ** drei Lebensleisten und der Spieler beginnt mit der zweiten, halb gefüllten Lebensleiste beim Start **. Spieler (gewinnt / verliert) die Lebensleiste durch die Nachricht (Stream Up! / Stream Down!) Für (Füllt die Lebensleiste voll / verbraucht alle HP in dieser Lebensleiste). Nachdem die Nachricht angezeigt wurde, zeigt das Spiel im Hintergrund eine Animation, die den Schwierigkeitsgrad (lauter / leiser) und eine halb gefüllte Lebensleiste anzeigt. Wenn ein Spieler ein "Stream Down!" Erhält, wird dem Spieler vorübergehend Unsichtbarkeit verliehen (da in der aktuell verwendeten Lebensleiste keine Gesundheit mehr vorhanden ist), bis die Animation abgeschlossen ist, in der die Lebensleiste durch eine zur Hälfte gefüllte aus der vorherigen ersetzt wird Schwierigkeit.
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
Im **Stream-Modus** erhält der Spieler jedoch **drei Lebensleisten und der Spieler beginnt mit der zweiten, halb gefüllten Lebensleiste beim Start**. Spieler (gewinnt / verliert) die Lebensleiste durch die Nachricht (Stream Up! / Stream Down!) Für (Füllt die Lebensleiste voll / verbraucht alle HP in dieser Lebensleiste). Nachdem die Nachricht angezeigt wurde, zeigt das Spiel im Hintergrund eine Animation, die den Schwierigkeitsgrad (lauter / leiser) und eine halb gefüllte Lebensleiste anzeigt. Wenn ein Spieler ein "Stream Down!" Erhält, wird dem Spieler vorübergehend Unsichtbarkeit verliehen (da in der aktuell verwendeten Lebensleiste keine Gesundheit mehr vorhanden ist), bis die Animation abgeschlossen ist, in der die Lebensleiste durch eine zur Hälfte gefüllte aus der vorherigen ersetzt wird Schwierigkeit.
```

File: wiki/osu!stream/de.md [hunk 13]
```diff
@@ -0,0 +1,254 @@
...
+
+Im ** Expertenmodus ** wird eine vollständige Lebensleiste ausgegeben **.
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
Im **Expertenmodus wird eine vollständige Lebensleiste ausgegeben**.
```

File: wiki/osu!stream/de.md [hunk 14]
```diff
@@ -0,0 +1,254 @@
...
+[Os-Pause.png]: ./img/Os-Pause.png "Pause menu"
+[Os-FM.png]: ./img/Os-FM.png "Menü fehlgeschlagen"
+[Os-SC.png]: ./img/Os-SC.png "Stage cleared!"
+[Os-Grade.jpg]: ./img/Os-Grade.jpg "Notenbildschirm"
+[Os-O1.png]: ./img/Os-O1.png "Optionsbildschirm 1"
+[Os-O2.png]: ./img/Os-O2.png "Optionsbildschirm 2"
+
+# osu! stream
+
+**[Klicken Sie hier, um zur Indexseite für externe Ports zurückzukehren](../"Indexseite für externe Ports")**
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
**[Klicken Sie hier, um zur Indexseite für externe Ports zurückzukehren](../ "Indexseite für externe Ports")**
```

File: wiki/osu!stream/de.md [hunk 15]
```diff
@@ -0,0 +1,254 @@
...
+
+![osu! stream logo][osu! stream logo]
+
+** osu! stream ** ist eine Version von osu! Von Grund auf neu gebaut, speziell für Touch-fähige Geräte. Es ist derzeit nur für Apples Plattform [iOS](http://en.wikipedia.org/wiki/IOS) verfügbar, jedoch für eine Plattform [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) ) Hafen ist geplant. Es führt ein brandneues Gameplay-System wie den Haltekreis und verbundene Beats ein.
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
**osu! stream** ist eine Version von osu! Von Grund auf neu gebaut, speziell für Touch-fähige Geräte. Es ist derzeit nur für Apples Plattform [iOS](http://en.wikipedia.org/wiki/IOS) verfügbar, jedoch für eine Plattform [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) ) Hafen ist geplant. Es führt ein brandneues Gameplay-System wie den Haltekreis und verbundene Beats ein.
```

File: wiki/osu!stream/de.md [hunk 16]
```diff
@@ -0,0 +1,254 @@
...
+
+Es unterstützt kein klassisches osu! Beatmaps. Stattdessen werden Karten von einem speziellen Team von Kartierern erstellt, um die neuen Elemente zu nutzen und strengere Standards einzuhalten. Sowohl kostenlose als auch kostenpflichtige Karten sind im "Store" erhältlich.
+
+[osu! stream wurde zusammen mit einem Interview in osu! month \ # 5 ausgearbeitet.](https://osu.ppy.sh/community/forums/topics/59924)
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
[osu! stream wurde zusammen mit einem Interview in osu! month \# 5 ausgearbeitet.](https://osu.ppy.sh/community/forums/topics/59924)
```

File: wiki/osu!stream/de.md [hunk 17]
```diff
@@ -0,0 +1,254 @@
...
+- Gehen Sie vom iDevice zum "App Store"
+- Suchen Sie nach osu! Stream
+- Drücken Sie und installieren Sie osu! Stream
+  - Wenn Sie aufgefordert werden, die Apple ID und das Passwort einzugeben, tun Sie dies bitte.
+- osu! stream kann abgespielt werden, nachdem der Download und die automatische Installation abgeschlossen sind.
+  - Es wird empfohlen, auch den Abschnitt "Hinzufügen von Beatmaps" zu lesen, um die neuesten kostenlosen Songs zu erhalten. Der aktuelle Download enthält nur 2 Songs.
+
+### USB-Methode
+
+** Auf dem Gerät muss [iTunes](https://www.apple.com/itunes/) installiert sein. **
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
**Auf dem Gerät muss [iTunes](https://www.apple.com/itunes/) installiert sein.**
```

File: wiki/osu!stream/de.md [hunk 18]
```diff
@@ -0,0 +1,254 @@
...
+- Verbundene Beats sind nicht nur auf "Hit Circle-Hit Circle" beschränkt, sondern können "Hit Circle-Slider", "Hold Circle-Slider" usw. sein.
+- Verbundene Beats können identifiziert werden, indem eine gerade weiße Linie mit einem anderen Objekt verbunden wird.
+
+##### Hold circle
+
+![Hold circle 1][PC.jpg]
+
+_Hold circle 1_
+
+-
```
Comment (MegaApplePi) [inline wiki/osu!stream/de.md]:
```suggestion
---
```

This is a horizontal bar, needs at least three hyphens (`-`)

**Comment by MegaApplePi** [comment]
@Firepe You can group all of the suggested changes by going here
image

Then click on this button
image

Then commit using this
image

File: wiki/osu!stream/de.md (hunk 8 above)
Comment (Firepe) [reply wiki/osu!stream/de.md]: Tried to fix. Please let me know of any errors.

**Comment by Firepe** [comment]
ok sorry

**Comment by Firepe** [comment]
thank you!

**Pull request closed without merging**